docs/operator-manual/argocd-cmd-params-cm.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: argocd-cmd-params-cm
+  labels:
+    app.kubernetes.io/name: argocd-cmd-params-cm
+    app.kubernetes.io/part-of: argocd
+data:
+  # Repo server address. (default "argocd-repo-server:8081")
+  repo.server: "argocd-repo-server:8081"
+  
+  # Dex server address (default "http://argocd-dex-server:5556")
+  dex.server: "http://argocd-dex-server:5556"
+
+  # Redis server hostname and port (e.g. argocd-redis:6379)
+  redis: "argocd-redis:6379"
```

Comment:
Likewise to what @alexmt said, if we use `redis.db` below, we should also name this not just `redis`, but `redis.server`